esm/interface/event-target.js

```diff
@@ -5,4 +5,22 @@ import EventTarget from '@ungap/event-target';
 /**
  * @implements globalThis.EventTarget
  */
-export {EventTarget};
+class DOMEventTarget extends EventTarget {
+  getParent() {
+    return null;
+  }
+
+  dispatchEvent(event) {
+    super.dispatchEvent(event);
```

Comment:
I think the correct way here is `const dispatched = super.dispatchEvent(event);` so that you do anything else *only* if `dispatched` is `true`, and you return `dispatched` at the end.